.github/workflows/if-docker-pr-testing.yml

```
@@ -16,7 +16,24 @@ jobs:
     runs-on: ubuntu-latest
     
     steps:
-      - if: "github.event.pull_request.draft == false &&!((github.actor == 'asyncapi-bot' && startsWith(github.event.pull_request.title, 'ci: update global workflows')) || (github.actor == 'asyncapi-bot' && startsWith(github.event.pull_request.title, 'chore(release):')) || (github.actor == 'allcontributors' && startsWith(github.event.pull_request.title, 'docs: add')))"
+      - if: >
+          !(github.event.pull_request.draft &&
+            (github.actor == 'asyncapi-bot' && !(
+              startsWith(github.event.pull_request.title, 'ci: update global workflows') || 
```

the `!` after `'asyncapi-bot' &&` should be removed.

```suggestion
          !github.event.pull_request.draft && !(
            (github.actor == 'asyncapi-bot' && (
              startsWith(github.event.pull_request.title, 'ci: update global workflows') || 
```